src/sagemaker/jumpstart/utils.py

```diff
@@ -271,26 +271,41 @@ def add_jumpstart_tags(
         training_script_uri (Optional[str]): S3 URI for training script tarball.
             (Default: None).
     """
-
+    warn_msg = (
+        "The uri (%s) is a pipeline variable which is parsed only in execution time. "
```

Comment:
parsed --> resolved?

Reply:
Thus skip adding custom tags to JumpStart models for this uri in SDK.

--->

Thus skip adding custom tags to JumpStart models for this uri during compile time. ?

Reply:
These comments might seem nitty but let's treat the warnings as docs as far as readability. 

Suggested:
`The URI (%s) is a pipeline variable which is only parsed (interpreted?) at execution time. As a result, the JumpStart resources will not be tagged.`

Reply:
Good call out! Updated in the new commit.